README.md

```diff
@@ -52,12 +53,12 @@ See [documentation site][GitHub Pages site].
 ### Modules
 
 #### Frame
-Frame was so cool it moved to [its own project](https://github.com/UW-Madison-DoIT/uw-frame).
+uPortal Home is a [uPortal App-Framework project](https://github.com/uPortal-Project/uportal-app-framework).
```

Comment:
Good to know. :+1: 
I was under the mistaken impression that the reverse was true.